### DISCUSSION TITLE
Defining multiple entities

### DISCUSSION THREAD

**Discussion by rapetti-carolina** [feast-dev/feast#4321] [Q&A] [answered]
Hi! I would like to have multiple entities defined and use them in different Feature Views. Currently I have one entity defined as 
`account = Entity(
    name="id_account", join_keys=["id_account"], value_type=ValueType.STRING, description="The account id")`

and I want to add another entity like
`other_entity = Entity(
    name="other_id", join_keys=["other_id"], value_type=ValueType.STRING, description="Different id, identifies a person")`

The question is: Is it correct to define the entities that way, one just behind the other? Or how should I define them?

**Comment by franciscojavierarceo** [comment, marked as answer]
It's fine to have separate entities and join them where you feel is appropriate, depending upon the context.
